.travis.yml

```diff
@@ -4,8 +4,6 @@ os: linux
 python:
     - "3.6"
     - "3.5"
-    - "3.4"
-    - "2.7"
```

Comment:
Numpy as already deprecated python 2 and will stop supporting it in ~6 months. I'm aware that there are many users that still use python 2 but I feel like that should be strongly discouraged, considering that it's been almost a decade since numpy started supporting python 3.

Reply:
I agree, but if you want this PR to be merged before January next year, then it needs to have python 2 support.